internal/blueprint/customizations.go

```diff
@@ -107,8 +107,7 @@ type ServicesCustomization struct {
 }
 
 type OpenSCAPCustomization struct {
-	DataStream string `json:"datastream,omitempty" toml:"datastream,omitempty"`
```

Comment:
The official RH documentation refers to the `datastream` option. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/assembly_creating-pre-hardened-images-with-image-builder-openscap-integration_composing-a-customized-rhel-system-imag

While I agree with hardcoding the path, it would be IMO better for backward compatibility to still keep the option and use the hardcoded path if none was provided by the user. Removing it at this point is IMO too late. We can still decide to not expose it in the Cloud API or even limit it only on the image builder API level...

Reply:
Yeah, probably don't want to break backwards compatibility, my suggestion was meant to be to only restrict it in the cloud api as that is new.

Reply:
My bad, I misunderstood that. Will fix